tests/build/build_templates.go

```diff
@@ -599,13 +599,15 @@ var _ = framework.BuildSuiteDescribe("Build templates E2E test", Label("build",
 
 						// Refresh our copy of the PipelineRun for latest results
 						pr, err = kubeadminClient.TektonController.GetPipelineRun(pr.Name, pr.Namespace)
+						GinkgoWriter.Printf("The PipelineRun %s in namespace %s has status.conditions: \n%v\n", pr.Name, pr.Namespace, pr.Status.Conditions)
```

Comment:
Do you have an example of when this line is printed?

I wonder if `%#v` would make it clearer?

Reply:
```$ ./bin/e2e-appstudio --ginkgo.focus="Build templates E2E test" --ginkgo.vv
...

Reply:
after take the comment `%#v`, it is more clear.
``` 
 The PipelineRun ec-integration-test-scenario-run-p2jpc in namespace build-e2e-bbpg-tenant has status.conditions: 
  v1.Conditions{apis.Condition{Type:"Succeeded", Status:"False", Severity:"", LastTransitionTime:apis.VolatileTime{Inner:time.Date(2024, time.July, 26, 14, 30, 39, 0, time.Local)}, Reason:"Failed", Message:"Tasks Completed: 1 (Failed: 1, Cancelled 0), Skipped: 0"}}
  The TaskRun ec-integration-test-scenario-run-p2jpc-verify of PipelineRun ec-integration-test-scenario-run-p2jpc  has status.conditions: 
  v1.Conditions{apis.Condition{Type:"Succeeded", Status:"False", Severity:"", LastTransitionTime:apis.VolatileTime{Inner:time.Date(2024, time.July, 26, 14, 30, 39, 0, time.Local)}, Reason:"Failed", Message:"\"step-assert\" exited with code 1"}}